asdf.sh

```diff
@@ -8,7 +8,9 @@ fi
 
 export ASDF_DIR
 ASDF_DIR="$(cd "$(dirname "$current_script_path")" &> /dev/null || exit 1; pwd)"
-export PATH="${ASDF_DIR}/bin:${ASDF_DIR}/shims:$PATH"
+
+[[ ":$PATH:" != *":${ASDF_DIR}/bin:"* ]] && PATH="${ASDF_DIR}/bin:$PATH"
+[[ ":$PATH:" != *":${ASDF_DIR}/shims:"* ]] && PATH="${ASDF_DIR}/shims:$PATH"
```

Comment:
The checks for `/bin` and `/shims` are separate entries as I've noticed over the years that my PATH ordering gets shuffled around which one can only assume means other tools are editing and inserting between items in the PATH list.